pkg/orchestrator/endpoint.go

```diff
@@ -107,14 +155,39 @@ func (e *BaseEndpoint) SubmitJob(ctx context.Context, request *SubmitJobRequest)
 	}, nil
 }
 
+func (e *BaseEndpoint) DiffJob(ctx context.Context, request *DiffJobRequest) (_ *DiffJobResponse, err error) {
+	job := request.Job
+	job.Normalize()
```

Comment:
If the Job Normalization or Transformation creates the ID of the JOb, try to move it to create Job for better isolation

Reply:
Issue `ENG-855`